pandas/tests/indexes/common.py

```diff
@@ -933,3 +933,10 @@ def test_empty(self):
         index = self.create_index()
         assert not index.empty
         assert index[:0].empty
+
+    @pytest.mark.parametrize('how', ['outer', 'inner', 'left', 'right'])
```

Comment:
do mean mean add the code that generated the issue as a test (joining two randomly generated 5x5 DataFrames)?  I don't really understand what you're asking me to do.